app/src/components/OrderBookSnapshot.js

```diff
@@ -27,69 +30,67 @@ class OrderBookSnapshot extends Component {
-        }, () => { this.handleChangeDateTime(); });
-    };
-
-    /**
-     * @desc Handles the time change when sliding the time Slider
-     * @param event The event object that caused the call
-     * @param value The new time value that represents the nanoseconds between 12 am and the chosen time of day.
-     */
-    handleChangeTime = (event, value) => {
-        let selectedTimeNano = parseInt(value);
-
-        this.setState({
-            selectedTimeNano: selectedTimeNano,
-            selectedTimeString: nanosecondsToString(selectedTimeNano),
-        });
-    };
-
-    /**
-     * @desc Handles the time change when the user stops sliding the time Slider
-     * @param event The event object that caused the call
-     * @param value The new time value that represents the nanoseconds between 12 am and the chosen time of day
-     */
-    handleCommitTime = (event, value) => {
-        const { selectedDateNano } = this.state;
-
-        let selectedTimeNano = parseInt(value);
-        let selectedDateTimeNano = selectedTimeNano + selectedDateNano;
-
-        this.setState({
-            selectedTimeNano,
-            selectedTimeString: nanosecondsToString(selectedTimeNano),
-            selectedDateTimeNano
-        }, () => this.handleChangeDateTime());
-    };
-
-    /**
-     *  @desc Updates the selectedDateTimeNano state variable when there is a
-     *  change in the date or when the user stops sliding the time Slider
-     */
-    handleChangeDateTime = () => {
-        const { selectedDateTimeNano, selectedDateNano } = this.state;
-
-        if (selectedDateTimeNano !== 0 && selectedDateNano !== 0) {
-            OrderBookService.getOrderBookPrices(SNAPSHOT_INSTRUMENT, selectedDateTimeNano)
-                .then(response => {
-                    this.setState({
-                        asks: response.data.asks,
-                        bids: response.data.bids,
-                    });
-                })
-                .catch(err => {
-                    console.log(err);
-                });
-        }
+    /**
+     * @desc Handles the date change for the TextField date picker
+     * @param event The event object that caused the call
+     */
+    handleChangeDate = (event) => {
+        let selectedDateNano = parseInt(dateStringToEpoch(event.target.value + ' 00:00:00'));
+
+        this.setState({
+            selectedDateNano,
+        }, () => { this.handleChangeDateTime(); });
+    };
+
+    /**
+     * @desc Handles the time change when sliding the time Slider
+     * @param event The event object that caused the call
+     * @param value The new time value that represents the nanoseconds between 12 am and the chosen time of day.
+     */
+    handleChangeTime = (event, value) => {
+        let selectedTimeNano = parseInt(value);
+
+        this.setState({
+            selectedTimeNano: selectedTimeNano,
+            selectedTimeString: nanosecondsToString(selectedTimeNano),
+        });
+    };
+
+    /**

```

Comment:
why is all this considered different ?

Reply:
I think this is reverting the line endings. I just opened my pr, and I have a huge diff like this too.